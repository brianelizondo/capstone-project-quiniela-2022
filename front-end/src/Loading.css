.Loading{
    width: 100%;
    font-weight: bold;
    text-align: center;
    text-transform: uppercase;
    margin-top: 150px;
}