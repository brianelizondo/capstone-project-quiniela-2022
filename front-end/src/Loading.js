import React from 'react';
import { Spinner } from 'react-bootstrap';
import './Loading.css';

function Loading(){
    return (
        <div className="Loading">
            <Spinner animation="border" role="status" variant="light"></Spinner>
        </div>
    );
}

export default Loading;