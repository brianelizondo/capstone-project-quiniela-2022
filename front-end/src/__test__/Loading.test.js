import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

import Loading from '../Loading';

// Smoke Test
describe("Smoke Test", () => {
    it("renders 'Loading' component without crashing", () => {
        render(<Loading />);
    });
});

// Snapshot Tests
describe("Snapshot Test", () => {
    it("'Loading' matches snapshot", () => {
        const {asFragment} = render(<Loading />);
        expect(asFragment()).toMatchSnapshot();
    });
});