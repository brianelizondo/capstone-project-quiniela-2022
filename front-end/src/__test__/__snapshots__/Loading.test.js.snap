// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test 'Loading' matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Loading"
  >
    <div
      class="spinner-border"
      role="status"
    />
  </div>
</DocumentFragment>
`;
